grails-app/services/org/pih/warehouse/product/ProductService.groovy

```diff
@@ -1272,6 +1280,7 @@ class ProductService {
             where product.active = 1 and (${terms.collect {"""
                 lower(product.name) like '%${it}%' 
                 or lower(product.product_code) like '${it}%' 
+                or (synonym.synonym_type_code = 'DISPLAY_NAME' and synonym.name like '%${it}%')
```

Comment:
Could we inject here the enum value instead of hard-coding the `DISPLAY_NAME`?

Reply:
yeah I wasn't really sure what would be the best choice since this query expects a string so either a 
`DISPLAY_NAME` or `${SynonymTypeCode.DISPLAY_NAME.name()}`
but looking at it again I think the `enum.*.name()` makes a bit more sense
so I'll update it

Reply:
actually I don;t even need to `*.name()` since we already have 
```
   String toString() {
        return name()
    }
```
so that's great 
thanks @kchelstowski

Reply:
since `toString()` is overrided I don't think you even have to add `.name()`